### PR TITLE
ui/components/label: fix center top offset

### PR DESCRIPTION
**Pull request by NickeZ**
The labels size was incorrectly calculated when updating the label

### CODE REVIEW

File: src/ui/components/label.c [hunk 1]
```diff
@@ -175,6 +177,28 @@ static void _cleanup(component_t* component)
...
+
+    UG_FontSetVSpace(2);
+    UG_FontSelect(data->font);
+    if (data->scrollable) {
+        UG_MeasureStringNoBreak(&(label->dimension.width), &(label->dimension.height), data->text);
+        if (label->dimension.width < SCREEN_WIDTH) {
+            // Do not scroll if text already fits in the screen
+            data->scrollable = false;
+        }
+    } else if (
```
Comment (benma) [inline src/ui/components/label.c]: else if has no condition?

File: src/ui/components/label.c [hunk 2]
```diff
@@ -39,14 +39,16 @@ typedef struct {
     int16_t text_position_last;
 } data_t;
 
+static void _measure_label_dimensions(component_t* label);
```
Comment (benma) [inline src/ui/components/label.c]: could also move the function here instead.

File: src/ui/components/label.c [hunk 3]
```diff
@@ -175,6 +177,28 @@ static void _cleanup(component_t* component)
     ui_util_component_cleanup(component);
 }
 
+void _measure_label_dimensions(component_t* label)
```
Comment (benma) [inline src/ui/components/label.c]: needs `static` too I think

File: src/ui/components/label.c (hunk 1 above)
Comment (NickeZ) [reply src/ui/components/label.c]: I just moved the code. I'll fix it

File: src/ui/components/label.c (hunk 2 above)
Comment (NickeZ) [reply src/ui/components/label.c]: sure

File: src/ui/components/label.c (hunk 3 above)
Comment (NickeZ) [reply src/ui/components/label.c]: nope not allowed.

File: src/ui/components/label.c (hunk 1 above)
Comment (NickeZ) [reply src/ui/components/label.c]: I think you are mis-seeing a left parenthesis as a left bracket.

**Pull request merged**